elfi/methods/parameter_inference.py

```diff
@@ -936,54 +935,87 @@ def _report_batch(self, batch_index, params, distances):
             str += "{}{} at {}\n".format(fill, distances[i].item(), params[i])
         logger.debug(str)
 
-    def plot_state(self, **options):
-        """Plot the GP surface
-        
-        This feature is still experimental and currently supports only 2D cases.
+    def plot_acq_points(self):
```

Comment:
I was planned that the actual code in the earlier version of `plot_state` would also be moved away from here. If you can do that at the same time, it would be great.